bubuku/controller.py

```diff
@@ -25,6 +25,19 @@ def can_run_at_exit(self) -> bool:
 
 
 class Check(object):
+
+    check_interval_s = 5
+    __last_check_timestamp_s = 0
```

Comment:
It's a class property, are you sure to add this property statically to all checks?